### PR TITLE
Fixed power netboot image links

### LINKED ISSUE #3746: Error downloading ubuntu 16.04 ppc
Found a dead link while trying to download ubuntu 16.04 for POWER

Here's the message I get
The requested URL /releases/16.04.2/release/ was not found on this server.

Thanks
---
*Reported from: https://www.ubuntu.com/download/server/power*

### PR DESCRIPTION
**Pull request by pmahnke**
## Done

- Removed point releases for 14.04 and 16.04 netbook links
- Updated the [copy doc](https://docs.google.com/document/d/1e-vav_b54KlKsi92w3rDu9J14O0gpp8yAanEAIImlE8/edit#)

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [/download/server/power](http://0.0.0.0:8001/download/server/power)
- Run through the following [QA steps](https://github.com/canonical-webteam/practices/blob/master/workflow/qa-steps.md)
- See that the 14.04 and 16.04 netbook links go to a page that has the generic series netbook link

## Issue / Card

Fixes #3746

### CODE REVIEW

File: templates/download/server/power.html [hunk 1]
```diff
@@ -29,8 +29,8 @@ <h2>Netboot image</h2>
...
-          <li class="p-list__item"><a href="http://cdimage.ubuntu.com/netboot/{{ previous_lts_release_with_point }}/" onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Download', 'eventAction' : 'POWER netboot', 'eventLabel' : '16.04.2', 'eventValue' : undefined });">{{ previous_lts_release_with_point }} <abbr title="Long-term support">LTS</abbr>&nbsp;&rsaquo;</a></li>
-          <li class="p-list__item"><a href="http://cdimage.ubuntu.com/netboot/14.04.6/" onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Download', 'eventAction' : 'POWER netboot', 'eventLabel' : '14.04.6', 'eventValue' : undefined });">14.04.6&nbsp;&rsaquo;</a></li>
+          <li class="p-list__item"><a href="http://cdimage.ubuntu.com/netboot/16.04/" onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Download', 'eventAction' : 'POWER netboot', 'eventLabel' : '16.04', 'eventValue' : undefined });">16.04 <abbr title="Long-term support">LTS</abbr>&nbsp;&rsaquo;</a></li>
```
Comment (anthonydillon) [inline templates/download/server/power.html]: Why not use: `previous_lts_release`?

File: templates/download/server/power.html [hunk 2]
```diff
@@ -29,8 +29,8 @@ <h2>Netboot image</h2>
...
+          <li class="p-list__item"><a href="http://cdimage.ubuntu.com/netboot/14.04/" onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Download', 'eventAction' : 'POWER netboot', 'eventLabel' : '14.04', 'eventValue' : undefined });">14.04&nbsp;&rsaquo;</a></li>
```
Comment (anthonydillon) [inline templates/download/server/power.html]: As this is so old could we just link to [Older netbook versions](http://cdimage.ubuntu.com/netboot/)?

Comment (pmahnke) [reply templates/download/server/power.html]: good call, I removed all 14.04 stuff

File: templates/download/server/power.html (hunk 1 above)
Comment (pmahnke) [reply templates/download/server/power.html]: yes, done.

**Pull request merged**